src/models/spaces/populations/mouse-model/mice.ts

```diff
@@ -393,7 +393,7 @@ export function getMouseSpecies(model: MousePopulationsModelType) {
           }, {
             image: {
               path: "assets/curriculum/mouse/populations/sandrat-dark.png",
-              scale: 0.2,
+              scale: 0.16,
               anchor: {
```

Comment:
In general, I think these new scales are spot on.  I find the sim to be much more coherent and readable with these smaller scales.  👍